pkg/noderesourcetopology/score.go

```diff
@@ -63,15 +63,10 @@ func (tm *TopologyMatch) Score(ctx context.Context, state *framework.CycleState,
 		return framework.MaxNodeScore, nil
 	}
 
-	nodeTopology, ok := tm.nrtCache.GetCachedNRTCopy(nodeName, pod)
-
-	if !ok {
+	nodeTopology, err := tm.nrtCache.GetCachedNRTCopy(nodeName, pod)
+	if err != nil {
 		klog.V(4).InfoS("noderesourcetopology is not valid for node", "node", nodeName)
-		return 0, nil
-	}
-	if nodeTopology == nil {
-		klog.V(5).InfoS("noderesourcetopology was not found for node", "node", nodeName)
-		return 0, nil
+		return framework.MinNodeScore, nil
```

Comment:
again not sure (literally, need to check) that removing the `if nodeTopology == nil` guard has no ill effects in the flow.

Reply:
added this check again

Reply:
thanks. Could you please add back the log as well?

Reply:
:+1: